src/themes/light/custom.js

```diff
@@ -10,6 +10,9 @@ export const custom = {
       highlightLine: {
           stroke: 'hsla(205deg, 45%, 90%, 90%);'
       },
+      highlightLineBorder: {
+          stroke:  'hsla(204.762deg, 53%, 46%, 1);'
+      },
```

Comment:
You have added this for only primary line. I think this should also be added for secondary and danger line.
Also for different style you might have to use different stroke color.
Have you checked your changes with different line styles?

Reply:
Done